templates/bartholomew_sidebar.hbs

```diff
@@ -1,16 +1,28 @@
 <div class="menu-wrap is-narrow is-fixed-desktop" id="docsMenu">
     <aside class="menu">
         <p class="menu-label">
-            Using Bartholomew
+            Get Started
         </p>
         <ul class="menu-list">
             <li><a {{#if (active_content request.spin-path-info "/bartholomew/index" )}} class="active" {{/if}} href="{{site.info.base_url}}/bartholomew/">Introduction</a></li>
             <li><a {{#if (active_content request.spin-path-info "/bartholomew/quickstart" )}} class="active" {{/if}} href="{{site.info.base_url}}/bartholomew/quickstart">Quickstart</a></li>
-            <li><a {{#if (active_content request.spin-path-info "/bartholomew/templates" )}} class="active" {{/if}} href="{{site.info.base_url}}/bartholomew/templates">Templates</a></li>
             <li><a {{#if (active_content request.spin-path-info "/bartholomew/configuration" )}} class="active" {{/if}} href="{{site.info.base_url}}/bartholomew/configuration">Configuration</a></li>
+        </ul>
+
+        <p class="menu-label">
+            Tutorials
+        </p>
+        <ul class="menu-list">
+            <li><a {{#if (active_content request.spin-path-info "/bartholomew/markdown" )}} class="active" {{/if}} href="{{site.info.base_url}}/bartholomew/markdown">Writing Markdown</a></li>
+        </ul>
+
+        <p class="menu-label">
+            Tools
```

Comment:
I like the grouping here, but am not sold on "Tools" as a name.  "Sites" or "Building Sites" perhaps?

Though "building sites" sounds like construction areas I fear...!